docs/source/using/basic-commands.adoc

```diff
@@ -1,14 +1,14 @@
 include::variables.adoc[]
 
-= Managing {project}
+= Basic Commands
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 2
 
 toc::[]
 
-[[managing-minishift-overview]]
+[[basic-commands-overview]]
```

Comment:
-1 for naming this as`basic-commands` because topics like 
```
Runtime Options
Persistent Volumes
HTTP/HTTPS Proxies
Networking
```

are not basic commands IMO. Also I do not find it intuitive. If you want to have `basic commands` section we should split the doc in to two parts i.e. 1. Basic commands 2. Managing minishift.

Reply:
> -1 for naming this asbasic-commands because topics like

I want to get away from 'managing minishift'. I think this is a bad term regardless. I think the major parts of basic commands is exactly that - start/stop/config/ssh. 

Persistence volumes could be split out. Networking can be removed.

Reply:
Very good point, I happened to overlook this while reviewing the changes. I think it's also worth considering moving Experimental Features into its own topic under Using Minishift at some point.

I think that splitting the documentation between Basic Commands and Managing Minishift makes sense -- ultimately, these are different concepts.

Reply:
Except "Managing Minishift" is imo a misnomer anyway. Let's try to get away from it.

Reply:
Yes, it's not an ideal name, I agree. It's particularly vague.

Reply:
IMO the current change is already a step into the right direction. "Basic usage" might not be perfect, but it is better than "Managing Minishift". I have some more ideas and can do some further refinements as part of #952.  For now, I think there is enough value in this for a merge.

Reply:
@hferentschik @robin-owen I do not have a better suggestion for `Managing Minishift` at this moment. But I would suggest we split-up the actual content in to two parts. Changing page names might cause difficulty for users as they are familiar with certain structure or the docs. So we should only change the name when the new change makes the documentation more intuitive to navigate.

Reply:
> But I would suggest we split-up the actual content in to two parts. 

Into what? If you have start/stop/config/ssh under "basic", what do you place the other ones under? "Managing" does not work really, since the remaining sections would just be some informational tidbits.

I am intrigued why you think "Basic Commands" is so much worse than "Managing Minishift". In both cases you will have to actually at least look at the TOC to get a better understanding of what's covered. For me it sounds natural to start a topic like "Using Minishift" with "Basic Commands". Maybe it is the "Commands" part which you don't like? We could just use "Basics". The following topics would then be covering specific, more advanced features.

Reply:
For the time being, "Basic Usage" seems like an appropriate name to me.

Reply:
Basic Usage is reasonable

Reply:
Ok, "Basic Usage" it is then ;-) Let me adjust the PR...